backend/infrahub/menu/models.py

```diff
@@ -82,19 +82,19 @@ class MenuItem(BaseModel):
     def from_node(cls, obj: CoreMenuItem) -> Self:
         return cls(
             identifier=get_full_name(obj),
-            title=obj.label.value or "",
+            label=obj.label.value or "",
             icon=obj.icon.value or "",
             order_weight=obj.order_weight.value,
             path=obj.path.value or "",
-            kind=obj.get_kind(),
+            kind=obj.kind.value or "",
```

Comment:
Based on the `MenuItem` class and the `obj.kind.value or ""` it seems like kind, path and icon should perhaps be mandatory as strings but have `""` as the default value. Having said that I don't know if this would work in the frontend (.i.e. to send in a required blank value).